toc.yml

```diff
@@ -0,0 +1,7 @@
+- name: Home
+  href: README.md
+- name: Guides
+  href: Documentation/
+  homepage: Documentation/GettingStartedWithTheMRTK.md

```

Comment:
Would we prefer our home page to be the github landing page readme.md compared to getting started with the MRTK?